chainer_chemistry/models/gin.py

```diff
@@ -92,12 +102,92 @@ def __call__(self, atom_array, adj, is_real_node=None):
         for step in range(self.n_message_layers):
```

Comment:
self.n_update_layers to fix other issue.

Reply:
fixed!